video-presentation.py

```diff
@@ -230,6 +234,46 @@ def parsevalue(size, max_size):
         self.logger.debug("Masks after preparing: %s", parsed_masks)
         return parsed_masks
 
+    def create_video_previews(self, filename):
+        """Create mp4 and webm heavily compressed previews of the presentation to use in the previewer"""
+
+        # Let's not be greedy, use half available cores since we are probably in a docker container
+        # This could be done less crudely, we could leave this control to the container
+        encoding_threads = multiprocessing.cpu_count()
```

Comment:
This is not very resilient as @wpoely86 already mentioned. It might be wise to leverage some of https://stackoverflow.com/a/1006301